### PR TITLE
Tieline Default Credentials - Create CVE-2021-35336

### PR DESCRIPTION
**Pull request by pratikkhalane**
### Template / PR Information

-   Add **CVE-2021-35336**

-  References: https://pratikkhalane91.medium.com/use-of-default-credentials-to-unauthorised-remote-access-of-internal-panel-of-tieline-c1ffe3b3757c

### Template Validation

I've validated this template locally?

- [ ] YES

### Additional Details: 

Google Dork :  intext:2.6.4.8 - © 2019 tieline pty ltd

### Additional References:

- [Nuclei Template Creation Guideline](https://nuclei.projectdiscovery.io/templating-guide/)
- [Nuclei Template Matcher Guideline](https://github.com/projectdiscovery/nuclei-templates/wiki/Unique-Template-Matchers)
- [Nuclei Template Contribution Guideline](https://github.com/projectdiscovery/nuclei-templates/blob/master/.github/CONTRIBUTING.md)
- [PD-Community Discord server](https://discord.gg/projectdiscovery)

### CODE REVIEW

File: default-logins/Tieline/Tieline.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  author: Pratik Khalane
+  severity: high
+  description: Finding the  Tieline Admin Panels with default credentials.
+  reference: https://pratikkhalane91.medium.com/use-of-default-credentials-to-unauthorised-remote-access-of-internal-panel-of-tieline-c1ffe3b3757c
+  tags: Tieline,default-login
+
+#Payloads:
+
+#Username - admin
+#Password - password
```
Comment (toufik-airane) [inline default-logins/Tieline/Tieline.yaml:14]: Can you justify or clean up this existing code?

File: default-logins/Tieline/Tieline.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+requests:
+  - method: GET
+    path:
+      - '{{BaseURL}}/api/get_device_details'
+    headers:
+      User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:89.0) Gecko/20100101 Firefox/89.0
+      Referer: '{{BaseURL}}/assets/base/home.html'
+      Authorization: 'Digest username="admin", realm="Bridge-IT", nonce="d24d09512ebc3e43c4f6faf34fdb8c76", uri="/api/get_device_details", response="d052e9299debc7bd9cb8adef0a83fed4", qop=auth, nc=00000001, cnonce="ae373d748855243d"'
```
Comment (toufik-airane) [inline default-logins/Tieline/Tieline.yaml:23]: Can you give more context about the nonce and the cnonce hard coded value?

**Review by toufik-airane — CHANGES_REQUESTED** [review]
Dear @pratikkhalane,

Please find comments to address.

Thanks.

File: default-logins/Tieline/Tieline.yaml (hunk 1 above)
Comment (pratikkhalane) [reply default-logins/Tieline/Tieline.yaml:14]: This is just to give a reference of what are the default credentials for the people to read.

File: default-logins/Tieline/Tieline.yaml (hunk 2 above)
Comment (toufik-airane) [reply default-logins/Tieline/Tieline.yaml:23]: After investigation, you have to follow the authentication flow dictated by the server.
https://en.wikipedia.org/wiki/Digest_access_authentication (read the Example with explanation)

The Digest Authorization is not a one-way workflow like the Basic Authorization.
Unfortunately, the authentication flow requires extracting the WWW-Authenticate specified by the server.

cc @pratikkhalane @ehsandeep

**Comment by toufik-airane** [comment]
After investigation, we have to deal with an HTTP digest access authentication.
It requires further capabilities with the Nuclei engine.
Please take a look at the issue https://github.com/projectdiscovery/nuclei/issues/847.

We will keep you up to date to address this template.

**Review by toufik-airane — APPROVED** [review]
@princechaddha I've tested on a bunch of hosts and I confirm it works! 👍

**Comment by pratikkhalane** [comment]
Thank you😀

**Pull request merged**